### PR TITLE
Fixes #10211: If we change the status of "Display rule compliance columns", we need to refresh the page to be able to revert to previous value

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/10211

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala [hunk 1]
```diff
@@ -716,28 +716,38 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
     ( configService.send_server_metrics match {
       case Full(value) =>
         var sendMetrics = value
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala]: Can you rename it to "initSendMetrics"

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala [hunk 2]
```diff
@@ -716,28 +716,38 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
...
+        var x = value
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala]: to currentSendMetrics

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala [hunk 3]
```diff
@@ -753,26 +763,29 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
     ( configService.display_changes_graph() match {
       case Full(value) =>
         var displayGraphs = value
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala]: Same here, can you renamed the variables

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala [hunk 4]
```diff
@@ -797,26 +810,29 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
     ( configService.rudder_ui_display_ruleComplianceColumns() match {
       case Full(value) =>
         var displayColumns = value
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala]: Same here

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
rebase is not enough raphael, you need to to the merge like said in comment :)

**Comment by RaphaelGauthier** [comment]
OK, merging this PR

**Pull request merged**